.ci/set-milestone-on-referenced-issue.sh

```diff
@@ -17,7 +17,7 @@ echo "COMMITS_ARRAY=$COMMITS_ARRAY"
 
 echo "Extracting commit messages from commits array that match '^(Pull|Issue) #[0-9]+'."
 COMMIT_MESSAGES=$(echo "$COMMITS_ARRAY" | jq -r .[].message \
-  | grep -E "^(Pull|Issue) #[0-9]+")
+  | grep -E "^(Pull|Issue) #[0-9]+" || true)
```

Comment:
The solution was taken from here:
https://stackoverflow.com/a/61321426/9553927

grep returns an exit code 1 if it doesn't find lines that match the regex. In this case, we force it to return 0.

Reply:
I more like ` | cat` to avoid grep failures.

Reply:
@romani  ` | cat`  is not intuitive, we try to be Java friendly above all, `|| true` is more clear. Also - shellcheck does not like useless `cat`s.

Reply:
I thought we disabled such rule

Reply:
@romani That was for `wc -l`

Reply:
I am not an expert here ...... but ` | cat` works well in sequential greps.
may be 
`grep x || true | grep y || true | grep z || true`
vs
`grep x | cat | grep y | cat | grep z | cat`

At least in second you do not need to remember some nuances, cat just re-prints, so if we deal with string chain-pipe updates, it might be better.

lets keep "|| true".